apps/site/src/views/Schedule/Schedule.module.scss

```diff
@@ -0,0 +1,23 @@
+@use "bootstrap-utils" as bootstrap;
+
+$container-padding: 8rem;
+
+.schedule {
+	margin: 10rem 1rem;
+
+	@include bootstrap.media-breakpoint-up(md) {
+		margin: 10rem 7rem;
+	}
+
+	h2 {
+		text-align: center;
+	}
+
+	:global {
+		section.container {
+			// responsive padding
+			@include bootstrap.padding-top($container-padding);
+			@include bootstrap.padding-bottom($container-padding);
+		}
+	}
```

Comment:
Suggestion: if we're reusing this container padding scheme on all pages, we can consider including it in `globals.scss` or creating a mixin for it.

Reply:
Moved to `globals.scss`